content/docs/dvclive/api-reference/live/make_dvcyaml.md

```diff
@@ -0,0 +1,42 @@
+# Live.make_dvcyaml()
+
+Writes [DVC configuration](/doc/user-guide/project-structure/dvcyaml-files) for
+metrics, plots, and parameters to
+[`Live.dvc_file`](/doc/dvclive/api-reference/live#properties).
+
+```py
+def make_dvcyaml()
+```
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
+live.log_param("lr", 0.01)
+live.log_metric("acc", 0.9)
+live.log_sklearn_plot("confusion_matrix", [0, 0, 1, 1], [1, 0, 0, 1])
+live.make_dvcyaml()
+```
+
+## Description
+
+If `Live(dvcyaml=True)`, DVCLive will collect all the data logged in
```

Comment:
this is not very clear considering the example above, tbh ... it's True by default but we still call it explicitly .. does it mean that if it's false, then even that explicit call won't create a file?

Reply:
> does it mean that if it's false, then even that explicit call won't create a file?

That's correct. TBH I don't love that behavior but it matches what we do in `Live.make_report()`. Not sure that it's a use case I can imagine someone doing on purpose.

Reply:
yeah, quite unexpected behavior tbh (any specific reason for that? we are loosing flexibility bc of this also) ... but even if keep it, I would explicitly write about this.

Reply:
Agreed that it's unexpected. Changed in this PR and in https://github.com/iterative/dvclive/pull/447.

Reply:
thanks @dberenbaum !